package day38_exceptions;

public class TestPersonObjects {
    public static void main(String[] args) {

        Person person1 = new Person("Daniel", 22, 'K');
        System.out.println(person1);

      /*  try{
            person1.setAge( 190);
        }catch (RuntimeException e){
            e.printStackTrace();
        }
       */

        System.out.println(person1);





    }
}
